src/Access/UsersConfigAccessStorage.cpp

```diff
@@ -108,6 +111,24 @@ namespace
             user->auth_data = AuthenticationData{AuthenticationType::KERBEROS};
             user->auth_data.setKerberosRealm(realm);
         }
+        else if (has_certificates)
+        {
+            user->auth_data = AuthenticationData{AuthenticationType::CERTIFICATE};
+
+            /// Fill list of allowed certificates.
+            Poco::Util::AbstractConfiguration::Keys keys;
+            config.keys(certificates_config, keys);
+            user->auth_data.clearAllowedCertificates();
+            for (const String & key : keys)
+            {
+                if (key.starts_with("common_name"))
```

Comment:
ok,
i saw the trick with starts_with is used in the existing code and tried to be consistent with it

Reply:
Probably checking `if (key == "common_name" || key.starts_with("common_name["))` is better because if you check a weaker condition `if (key.starts_with("common_name"))` you will allow any variants started with `common_name`, for example `common_name123`.